common/terms.html

```diff
@@ -59,18 +59,17 @@
 
 <p>Terms imported from [[[RFC3987]]] [[RFC3987]]</p>
 <dl class="termlist" data-sort>
-  <dt><dfn data-cite="RFC3987#section-1.3" class="preserve">absolute IRI</dfn></dt><dd>
+  <dt><dfn data-cite="RFC3987#section-1.3" data-lt="Internationalized Resource Identifier" class="preserve">IRI</dfn></dt><dd>
     The absolute form of an <a>IRI</a> containing a <em>scheme</em> along with a <em>path</em>
     and optional <em>query</em> and <em>fragment</em> segments.</dd>
-  <dt><dfn data-cite="RFC3987#section-1.3" data-lt="IRI|Internationalized Resource Identifier" class="preserve"><abbr title="Internationalized Resource Identifier">IRI reference</abbr></dfn></dt><dd>
+  <dt><dfn data-cite="RFC3987#section-1.3" class="preserve"><abbr title="Internationalized Resource Identifier">IRI reference</abbr></dfn></dt><dd>
     Denotes the common usage of an <a>Internationalized Resource Identifier</a>.
-    An <a>IRI reference</a> may be <a data-lt="absolute IRI">absolute</a> or
-    <a data-lt="relative IRI">relative</a>.
-    However, the "IRI" that results from such a reference
-    only includes <a>absolute IRIs</a>; any <a>relative IRI</a> references are
-    resolved to their absolute form.</dd>
-  <dt><dfn data-cite="RFC3987#section-6.5" class="preserve">relative IRI</dfn></dt><dd>
-    A relative IRI is an <a>IRI</a> that is relative to some other <a>absolute IRI</a>,
+    An <a>IRI reference</a> may be absolute or
+    <a data-lt="relative IRI reference">relative</a>.
+    However, the "IRI" that results from such a reference only includes absolute <a>IRIs</a>;
+    any <a>relative IRI references</a> are resolved to their absolute form.</dd>
```

Comment:
Shouldn't this be a little more contextualized? For example "JSON-LD processors will resolve all relative IRI references to their absolute form."

Reply:
This is basically a copy/paste from the definition in RFC3987, as it's not really our term to define. The `data-cite` will cause all references to just go there. The practice for terms that are "imported" from other specs is to basically just summarize their own definitions.